stream-chat-android-client/src/main/java/io/getstream/chat/android/client/api/models/QuerySort.kt

```diff
@@ -90,30 +61,6 @@ public class QuerySort<T : Any> {
         }
     }
 
-    private fun Any.getMemberPropertyOrExtra(name: String): Any? =
-        name.snakeToLowerCamelCase().let { fieldName ->
-            this::class.memberProperties
-                .firstOrNull { it.name == fieldName }
-                ?.also {
-                    logger.d { "Could find a member for property: $fieldName" }
-                }
-                ?.getter
-                ?.call(this)
-                ?: (this as? CustomObject)?.extraData?.get(name).also { extraDataObject ->
```

Comment:
(this as? CustomObject) is a check that will never pass, because `getMemberPropertyOrExtra` is always being called from `String`, so it is never a CustomObject.

Reply:
Hey @leandroBorgesFerreira,

I double-checked that and it seems like `this` is actually a `CustomObject`, however, I can see you covered the `extraData` case by adding it as a first pace to check:
```kotlin
comparableFromExtraData(o1, this) ?: fieldSearcher.findComparable(o1, this),
```


Proof it's a CustomObject:
<img width="1267" alt="Screenshot 2022-06-01 at 14 05 03" src="https://user-images.githubusercontent.com/1286516/171380184-a6e36db3-fa91-4ef9-82d8-a71245c15945.png">